app/models/account_syncer.rb

```diff
@@ -0,0 +1,53 @@
+class AccountSyncer
```

Comment:
I think for now I'll throw it in `app/models/account`, but I do think there could be value in a `syncable` concern eventually as it would be nice to be able to use `family.sync`, `user.sync`, and `account.sync`.  In other words, `account.sync` would sync the account, while `family.sync` would sync all accounts owned by the family (i.e. global syncing)